website/src/pages/404.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+
+function ErrorPage() {
+  const context = useDocusaurusContext();
+  const { siteConfig = {} } = context;
+
+  function getTrackingScript() {
+    if (!siteConfig.gaTrackingId) {
+      return null
```

Comment:
there is no such field

https://v2.docusaurus.io/docs/docusaurus.config.js

Reply:
btw why do you need 404.js ? docusaurus handle it for you

Reply:
Docusaurus is returning a blank content for 404. Don't know why.

Reply:
Try https://v2.docusaurus.io/sdasjasad

Reply:
But why https://redux-starter-kit.netlify.com/fdsafdsf is showing a blank content?
I tried removing src/pages/404.js also. But no use.

Reply:
oh its because redux route basepath is empty so it routes to upper level docpage. Its equivalent to https://v2.docusaurus.io/docs

Seems like a bug

Reply:
Did adding 404.js solve the issue though ?

Reply:
This is fixed in next alpha release. 

You can safely remove the 404.js here

Reply:
Adding 404.js did not resolve the issue.
> oh its because redux route basepath is empty so it routes to upper level docpage. Its equivalent to https://v2.docusaurus.io/docs
Sorry. I didn't get this part.

Reply:
It's a react router concept. You can ignore it for now.

But just to share knowledge
```js
// routes.js
export default [
  {
    path: "/docs",
    component: ComponentCreator("/docs"),
    routes: [
      {
        path: "/docs/advanced-themes",
        component: ComponentCreator("/docs/advanced-themes"),
        exact: true
      },
      {
        path: "/docs/advanced-plugins",
        component: ComponentCreator("/docs/advanced-plugins"),
        exact: true
      },
    ]
  },
  {
    path: "*",
    component: ComponentCreator("*")
  }
];
```
imagine above scenario. It will match ;/docs/;

For redux case, it;s not `/docs` but `/` that's why you dont have 404 but an empty page, its rendering the parent nested route